app/components/CacheAnalysis.vue

```diff
@@ -131,86 +279,268 @@ onUnmounted(() => {
       </dt>
       <dd />
 
-      <dt>Cacheable</dt>
-      <dd>{{ cacheAnalysis.cacheControl.isCacheable ? "✅" : "❌" }}</dd>
+      <dt
+        class="data-key"
+        :class="{ 'key-highlighted': isKeyHovered('Cacheable') }"
+        @mouseenter="handleDataKeyHover('Cacheable', cacheAnalysis.cacheControl.isCacheable)"
+        @mouseleave="handleDataKeyLeave"
+      >
+        Cacheable
+      </dt>
+      <dd
+        class="data-value"
+        :class="{
+          'key-highlighted': isKeyHovered('Cacheable'),
+          'value-matching': isKeyHovered('Cacheable') && isValueMatching(cacheAnalysis.cacheControl.isCacheable),
+          'value-different': isKeyHovered('Cacheable') && !isValueMatching(cacheAnalysis.cacheControl.isCacheable),
+        }"
+      >
+        {{ cacheAnalysis.cacheControl.isCacheable ? "✅" : "❌" }}
+      </dd>
 
       <template v-if="cacheAnalysis.cacheControl.age">
-        <dt>Age</dt>
-        <dd :title="formatHumanSeconds(cacheAnalysis.cacheControl.age)">
+        <dt
+          class="data-key"
+          :class="{ 'key-highlighted': isKeyHovered('Age') }"
+          @mouseenter="handleDataKeyHover('Age', cacheAnalysis.cacheControl.age)"
+          @mouseleave="handleDataKeyLeave"
+        >
+          Age
+        </dt>
+        <dd
+          class="data-value"
+          :class="{
+            'key-highlighted': isKeyHovered('Age'),
+            'value-matching': isKeyHovered('Age') && isValueMatching(cacheAnalysis.cacheControl.age),
+            'value-different': isKeyHovered('Age') && !isValueMatching(cacheAnalysis.cacheControl.age),
+          }"
+          :title="formatHumanSeconds(cacheAnalysis.cacheControl.age)"
+        >
           {{ formatSeconds(cacheAnalysis.cacheControl.age) }}
+          <span
+            v-if="isKeyHovered('Age') && !isValueMatching(cacheAnalysis.cacheControl.age) && getDelta(cacheAnalysis.cacheControl.age)"
+            class="delta"
+          >
+            ({{ getDelta(cacheAnalysis.cacheControl.age) }})
+          </span>
         </dd>
       </template>
 
       <template v-if="cacheAnalysis.cacheControl.date">
-        <dt>Date</dt>
-        <dd>
+        <dt
+          class="data-key"
+          :class="{ 'key-highlighted': isKeyHovered('Date') }"
+          @mouseenter="handleDataKeyHover('Date', cacheAnalysis.cacheControl.date)"
+          @mouseleave="handleDataKeyLeave"
+        >
+          Date
+        </dt>
+        <dd
+          class="data-value"
+          :class="{
+            'key-highlighted': isKeyHovered('Date'),
+            'value-matching': isKeyHovered('Date') && isValueMatching(cacheAnalysis.cacheControl.date),
+            'value-different': isKeyHovered('Date') && !isValueMatching(cacheAnalysis.cacheControl.date),
+          }"
+        >
           {{ formatDate(cacheAnalysis.cacheControl.date) }}
```

Comment:
I think this is the bug for the date deltas not showing. You aren't calling `getDelta()` and rendering stuff here?